src/streams/StreamScreen.js

```diff
@@ -59,29 +60,31 @@ class StreamScreen extends PureComponent<Props> {
     const { styles } = this.context;
 
     return (
-      <Screen title="Stream" padding>
-        <StreamCard stream={stream} subscription={subscription} />
-        <OptionDivider />
+      <Screen title="Stream">
+        <View style={styles.padding}>
```

Comment:
Why don't you apply this style to the `View` inside the `StreamCard`?

Reply:
The approaches I could have taken are:

* Apply the padding to the `View` inside `StreamCard`, and apply it in `StreamScreen` for the buttons.

* Apply padding to both `StreamCard` and the buttons inside `StreamScreen` .

I chose the second approach as it felt cleaner and there is no 'mystery padding' to the `StreamCard` component. (None of the other components similar to `StreamCard` have padding presently.)

Reply:
I think it actually makes sense for a component like `StreamCard` to contain its own padding. It really is part of the layout of that information (the icon, the stream name, the description) that it needs some appropriate padding around it.

If there are similar components that are relying on their environment to add padding, we might change those later. This one only has a single use site, so we don't need to worry about different usages of it having mismatched expectations.

Reply:
Okay, I've made these changes. Now the `View` inside `StreamCard` contains padding, and it is no longer wrapped in a `View` just for padding in `StreamScreen` .